attest/verifier/types/build.rs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2023 The MobileCoin Foundation
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    prost_build::compile_protos(&["attest.proto"], &["../../api/proto"])?;
```

Comment:
I know this deviates from other prost usages, but I think we should be moving towards this, where the prost type is built from the *.proto file, instead of relying on maintainers to ensure the prost attribute tag matches that in the *.proto file. This also allows for the prost types to be made up of simple byte vectors while leaving the _common_ type to have typed fields